lite/examples/posenet/ios/PoseNet/Extensions/CVPixelBufferExtension.swift

```diff
@@ -120,7 +120,6 @@ extension CVPixelBuffer {
   /// - Returns: The RGB data representation of the image buffer or `nil` if the buffer could not be
   ///     converted.
   func rgbData(
```

Comment:
Could you also update the doc comment above that mentions the `byteCount` parameter? Thanks.

Reply:
@yyoon 
Sure. Thanks for your quick reply.